source/clear-linux/tutorials/multi-boot/multi-boot-rhel.rst

```diff
@@ -0,0 +1,168 @@
+.. _multi-boot-rhel:
+
+Install Red Hat\* Enterprise Linux 7.4 Beta
+*******************************************
+
+#. Start the Red Hat installer and follow the prompts.
+
+#. At the :guilabel:`INSTALLATION SUMMARY` screen, choose
+   :guilabel:`INSTALLATION DESTINATION`. See Figure 1.
+
+   .. figure:: figures/multi-boot-rhel-1.png
+
+      Figure 1: Red Hat: Installation summary
+
+#. In the :guilabel:`Device Selection` section, select a drive on which to
+   install the OS. See Figure 2.
+
+   .. figure:: figures/multi-boot-rhel-2.png
+
+      Figure 2: Red Hat: Installation destination
+
+#. Under the :guilabel:`Other Storage Options` section, choose
+   :guilabel:`I will configure partitioning`. See Figure 2.
+
+#. Click :guilabel:`Done`.
+
+#. Under :menuselection:`New Red Hat Enterprise Linux 7.4 Installation --> New mount points will use the following partitioning scheme` section,
+   select :menuselection:`Standard Partition` from the drop down list. See
+   Figure 3.
+
+   .. figure:: figures/multi-boot-rhel-3.png
+
+      Figure 3: Red Hat: New partition scheme
+
+#. Create a new root partition.
+
+   #. Click the :menuselection:`+` button on the lower left corner.
+
+   #. Enter `/` and the new partition size. For this example, we specified 45
+      GB. See Figure 4.
+
+      .. figure:: figures/multi-boot-rhel-4.png
+
+         Figure 4: Red Hat: Create new root partition
+
+   #. Click :guilabel:`Add mount point`.
+
+#. Share the swap partition that was created by |CL|. See Figure 5.
+
+   #. Expand :guilabel:`Unknown`.
+
+   #. Select :guilabel:`swap / sda2`.
+
+   #. Select :guilabel:`Reformat`.
+
+   #. Click :guilabel:`Update Settings`.
+
+      .. figure:: figures/multi-boot-rhel-5.png
+
+         Figure 5: Red Hat: Configure swap partition
+
+#. Share the EFI partition that was created by |CL|. See Figure 6.
+
+   #. Expand :guilabel:`Unknown.`
+
+   #. Select :guilabel:`EFI System Partition / sda3`.
+
+   #. Under :guilabel:`Mount Point`, enter `/boot/efi`.
+
+   #. Click :guilabel:`Update Settings`.
+
+      .. figure:: figures/multi-boot-rhel-6.png
+
+         Figure 6: Red Hat: Configure EFI partition
+
+#. Click :guilabel:`Done`.
+
+#. Follow the remaining prompts to complete the installation of Red Hat.
+
+#. At this point, the ability to boot |CL| is lost because `Grub`
+   was set as the default boot loader. Follow these steps to make the |CL|
+   Systemd-Boot the default boot loader and add Red Hat as a boot option:
+
+   #. Boot into Red Hat.
+
+   #. Log in.
+
+   #. Get root privilege with the following command:
+
+      .. code-block:: console
+
+         $ sudo -s
```

Comment:
Remove "#. Get root privilege with the following command:" section.

Reply:
Text edits deferred to next phase of updates.